Dashboard/pages/recent_rides.py

```diff
@@ -0,0 +1,192 @@
+from typing import NoReturn
+import os
+import datetime
+import dash
+from dash import Dash, dcc, html, Output, Input, callback
+
+
+dash.register_page(__name__, path="/")
+
+
+layout = \
+    html.Div([
+        # Interval timers for updating graphs
+        dcc.Interval(
+             id="user-interval-component",
+             interval=1000,
+             n_intervals=0
+             ),
+        dcc.Interval(
+            id="riders-interval-component",
+            interval=60*1000,
+            n_intervals=0
+        ),
+        dcc.Interval(
+            id="interval-component",
```

Comment:
Rename this to be more specific, I believe it links to the get kafkha data callback